soh/soh/Enhancements/randomizer/randomizer_tricks.cpp

```diff
@@ -44,6 +44,7 @@ std::unordered_map<RandomizerTrick, RandomizerTrickObject> rtObjects = {
     //RT_OBJECT(RT_BUNNY_HOOD_JUMPS,                        RTVORMQ_BOTH,    RTAREA_GENERAL,                &advancedBase,               false,                                     "Bunny Hood Jumps", "Allows reaching locations using Bunny Hood's extended jumps."),
     //RT_OBJECT(RT_DAMAGE_BOOST_SIMPLE,                     RTVORMQ_BOTH,    RTAREA_GENERAL,                &advancedBase,               false,                                     "Simple damage boosts", "Allows damage boosts to reach Hyrule Castle guards, the bomb bag area in DC and the Gerudo Valley crate Piece of Heart. Can be combined with \"Simple hover boosts\" for reaching far distances."),
     //RT_OBJECT(RT_HOVER_BOOST_SIMPLE,                      RTVORMQ_BOTH,    RTAREA_GENERAL,                &advancedBase,               false,                                     "Simple hover boosts", "Allows equipping of hover boots when link is moving at high speeds to extend distance covered. Can be combined with \"Simple damage boosts\" for greater uses."),
+    RT_OBJECT(RT_EXPLODE_BEEHIVES,                          RTVORMQ_BOTH,    RTAREA_GENERAL,                &noviceBase,                 false,                                     "Explode Beehives", "Allows exploding beehives with bombchus"),
```

Comment:
Now that bombs are added I think the short name should also explain that the trick is only for bombchus (and maybe update the RT_EXPLOSIVE_BEEHIVES name since thats also a bit misleading)

Reply:
`RT_EXPLODE_BEEHIVES_BOMBCHUS`?

Reply:
id just go for RT_BOMBCHU_BEEHIVES personally

Reply:
Done.